bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java

```diff
@@ -194,9 +208,9 @@ public ReadModifyWriteRowRequest adapt(Increment increment) {
    * @return a {@link com.google.bigtable.v2.MutateRowRequest} object.
    */
   public MutateRowRequest adapt(Put put) {
-    MutateRowRequest.Builder builder = mutationAdapters.putAdapter.adapt(put);
-    builder.setTableName(getTableNameString());
-    return builder.build();
+    Mutation mutation = Mutation.create();
```

Comment:
Might make sense to use a RowMutation here and have the PutAdapter take a MutationApi interface